addons/test_portal/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+test_portal_0,xxx,model_test_portal_portal,base.group_system,1,1,1,1
```

Comment:
``access_test_portal_portal_system,access.test.portal.portal.system`` ? Just to be coherent with naming used in SM team (at least), aka: main model (access for ir.model.access), used for test_portal_portal_model, for group_system. Just some "top-bottom" naming we try to use.

Reply:
Those are in guidelines note.